BlueprintUI/Sources/Layout/LayoutWriter.swift

```diff
@@ -285,6 +285,24 @@ extension LayoutWriter {
                     )
                 }
             }
+            
+            func layout(in context: StrictLayoutContext, children: [StrictLayoutChild]) -> StrictLayoutAttributes {
+//                let options = StrictLayoutOptions(mode: .init(horizontal: .fill, vertical: .fill))
```

Comment:
Do we need to add a todo here and/or remove this commented code?

Reply:
Will remove — I think this cruft came along with the original prototype.